backend/app/models.py

```diff
@@ -36,3 +36,15 @@ class GerryDBTableBase(TimeStampMixin, SQLModel):
 class GerryDBTable(GerryDBTableBase, table=True):
     uuid: str = Field(sa_column=Column(UUIDType, unique=True))
     name: str = Field(nullable=False, unique=True)
+
+
+class Document(TimeStampMixin, SQLModel):
```

Comment:
Great to create the Document table. We'll likely add many more attributes as we go.
```suggestion
class Document(TimeStampMixin, SQLModel, table=True):
```